app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationViewModel.kt

```diff
@@ -157,12 +166,13 @@ class ConversationViewModel @Inject constructor(
 
     @Suppress("MagicNumber")
     fun sendAttachmentMessage(attachmentBundle: AttachmentBundle?) {
+        val sizeOf1MB = 1024 * 1024
```

Comment:
abit unclear to me what is happening here

Reply:
It is explained a bit below on `getAssetLimitInBytes` method. Free users can only send up to 25MB for assets vs the 100MB of users with a team. I added another comment to make it more comprehensive